packages/appkit/exports/react.ts

```diff
@@ -44,7 +44,7 @@ export type { AppKitOptions }
 // -- Hooks ------------------------------------------------------------
 export * from '../src/library/react/index.js'
 
-export function useAppKitNetwork() {
+export function useAppKitNetwork(): UseAppKitNetworkReturn {
```

Comment:
Do we need return type here ?

Reply:
I'd rather be explicit whenever possible so I think it's fine

Reply:
Every hook or function we create should be explicit as much as this IMO. This is also very helpful for documentation and versioning